intl/msg_hash_us.h

```diff
@@ -731,9 +731,9 @@ MSG_HASH(MENU_ENUM_LABEL_VALUE_INPUT_POLL_TYPE_BEHAVIOR_NORMAL,
 MSG_HASH(MENU_ENUM_LABEL_VALUE_INPUT_PREFER_FRONT_TOUCH,
       "Prefer Front Touch")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_INPUT_REMAPPING_DIRECTORY,
-      "Input Remapping Dir")
+      "Input Remapping (dir)")
```

Comment:
I hate this convention, I think it's dumb it shouldn't even include Dir or (dir), we're already under directory settings